### PR TITLE
Add investors section to website

### PR DESCRIPTION
**Pull request by Schmavery**
image

image

Also fixed up the H3.wide (the one with the wings) so that it doesn't have weird margins and doesn't wrap

### CODE REVIEW

File: frontend/website/src/InvestorsSection.re [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            ])
+          )>
+          <img
+            src=imageSrc
+            className=Css.(
+              style([
+                width(`rem(2.5)),
+                height(`rem(2.5)),
+                borderRadius(`percent(100.)),
+                marginRight(`em(1.)),
```
Comment (bkase) [inline frontend/website/src/InvestorsSection.re]: rem please

File: frontend/website/src/InvestorsSection.re [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+              merge([Style.Body.basic, style([color(Style.Colors.slate)])])
+            )>
+            {ReasonReact.string(name)}
+          </h4>
+        </div>,
+    };
+  };
+};
+
+let component = ReasonReact.statelessComponent("BuiltBy");
```
Comment (bkase) [inline frontend/website/src/InvestorsSection.re]: "Investors"

File: frontend/website/src/InvestorsSection.re [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            marginTop(`rem(3.)),
+            marginLeft(`auto),
+            marginRight(`auto),
+            justifyContent(`center),
+            gridTemplateColumns([`fr(1.), `fr(1.)]),
+            media(
+              Style.MediaQuery.notMobile,
+              [
+                // bs-css doesn't allow fr in minmax: https://github.com/SentiaAnalytics/bs-css/pull/124
+                `declaration((
```
Comment (bkase) [inline frontend/website/src/InvestorsSection.re]: you can use `unsafe("foo", "whatever")` also (this is what I've been using in grid)

File: frontend/website/src/InvestorsSection.re (hunk 1 above)
Comment (Schmavery) [reply frontend/website/src/InvestorsSection.re]: how did that one sneak in

**Pull request merged**